go.mod

```diff
@@ -139,57 +139,3 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
-
-replace k8s.io/api => k8s.io/api v0.28.0-rc.0
```

Comment:
Did the script remove this?

Reply:
No, I removed all the replace statements as we do for all other sidecars. Should these be kept there for external-provisioner?

Reply:
I think if the script is adding it let's just keep it

Reply:
Added these statements back in.